kolibri2zim/scraper.py

```diff
@@ -763,19 +799,34 @@ def run(self):
         self.output_dir.mkdir(parents=True, exist_ok=True)
 
         self.creator_lock = threading.Lock()
+        if not self.root_id:
+            logger.error("Missing root id")
+            return 1
+        if not self.title:
+            logger.error("Missing title")
+            return 1
+        if not self.description:
+            logger.error("Missing description")
+            return 1
+        if not self.author:
+            logger.error("Missing author")
+            return 1
+        if not self.publisher:
+            logger.error("Missing publisher")
+            return 1
         self.creator = Creator(
-            filename=self.output_dir.joinpath(self.fname),
+            filename=self.output_dir.joinpath(self.clean_fname),
             main_path=self.root_id,
             ignore_duplicates=True,
         )
         self.creator.config_metadata(
-            Name=self.name,
+            Name=self.clean_fname,
             Language="eng",
             Title=self.title,
             Description=self.description,
             Creator=self.author,
             Publisher=self.publisher,
-            Date=datetime.date.today().strftime("%Y-%d-%m"),
+            Date=datetime.datetime.now(datetime.UTC).strftime("%Y-%d-%m"),
```

Comment:
why?

Reply:
Because of [DTZ011](https://beta.ruff.rs/docs/rules/#flake8-datetimez-dtz). But I'm not sure this is really a good idea, I would rather deactivate this rule.